axelrod/tests/strategies/test_titfortat.py

```diff
@@ -466,6 +689,9 @@ def test_output_from_literature(self):
         Dilemma" Proc. Artif. Life 1996
 
         This test just ensures that the strategy is as was originally defined.
+
+        See https://github.com/Axelrod-Python/Axelrod/issues/1294 for another
+        discussion of this.
```

Comment:
Should this be a test similar to `test_specific_set_of_results` for `Gradual`? Run the whole tournament and compare the scores?

Reply:
Sure why not. :+1:

Reply:
Note that this will just repeat the doctest. (I'm fine with that.)

Reply:
Sure. It already does that but only for two strategies right?

We are testing the output of the paper so I suggest we include all of it, and it will follow the same format as the tests for `Gradual`.

Reply:
Done in 1c07e882